app.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const expressLayouts = require('express-ejs-layouts');
+const app = express()
```

Comment:
First require all of the modules at the top

Reply:
updated